hack/.ci/lib/e2e.sh

```diff
@@ -69,7 +75,31 @@ EOF
 }
 
 function gather-artifact-on-exit {
-  gather-artifacts || "Error gathering artifacts" > /dev/stderr
+  gather-artifacts "${ARTIFACTS}/must-gather" || "Error gathering artifacts" > /dev/stderr
+}
+
+# $1- kubeconfig ...
```

Comment:
```suggestion
# $...- kubeconfig(s)
```

Reply:
`$1` the items are not really all passed in the first arg but each of the items is an independent arg

Reply:
yeah I just didn't know how to put it there, thanks

Reply:
fixed